soccer/gameplay/tactics/positions/goalie.py

```diff
@@ -21,6 +21,10 @@ class Goalie(single_robot_composite_behavior.SingleRobotCompositeBehavior):
         robocup.Point(MaxX, constants.Robot.Radius + OFFSET))
```

Comment:
Are you sure that this is the correct goalie? I think that this goalie isn't used unless in extreme situations (I think during unknown states), I think the goalie you want to modify is [submissive_goalie](https://github.com/RoboJackets/robocup-software/blob/master/soccer/gameplay/tactics/positions/submissive_goalie.py), which runs under the [defense tactic](https://github.com/RoboJackets/robocup-software/blob/master/soccer/gameplay/tactics/defense.py).

Also see https://github.com/RoboJackets/robocup-software/issues/947 :( 

The reason for this is because it allows more advanced coordination between the goalie and the defense (which is needed for effective defense).

Reply:
I'm sure that this is the goalie that runs if you just start soccer and press the green check, beyond that I don't know.

Edit: Well now that you mention it submissive goalie does appear to be the one that runs when the other plays are running. I think I can modify that one using the exact same changes, as I think the relevant sections are identical.